packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -77,6 +79,26 @@ class IOSSimulatorUtils {
       );
     }).toList();
   }
+
+  Future<List<IOSSimulator>> getAvailableDevices() async {
+    if (!_xcode.isInstalledAndMeetsVersionCheck) {
+      return <IOSSimulator>[];
+    }
+
+    final List<SimDevice> available = await _simControl.getAvailableDevices();
+    return available
+        .map<IOSSimulator>((SimDevice device) {
+          return IOSSimulator(
```

Comment:
nit/optional: would be cool if we can also print the OS version since the default names are just the phone models and it's possible to download/instantiate more than 1 per model.

Reply:
Good idea.  The runtime is already included in the category, but it's not output in the `flutter emulator` list.